doc/scheduling.md

```diff
@@ -0,0 +1,180 @@
+# Scheduling in TempestaFW: User Guide
```

Comment:
Github wiki is more than simple markdown pages, see example here: https://github.com/opencomputeproject/onie/wiki   

Seems that it have separate repo, would know more in next couple of days